Makefile.am

```diff
@@ -372,6 +373,7 @@ EXTRA_DIST = \
 	tests/metrics/network/network-metrics-cpu-consumption.sh.in \
 	tests/metrics/network/network-metrics-memory-rss-1g.sh.in \
 	tests/metrics/network/network-metrics-memory-pss.sh.in \
+	tests/metrics/network/network-metrics-memory-pss-1g.sh.in \
```

Comment:
This new test needs to be added to https://github.com/01org/cc-oci-runtime/blob/master/tests/metrics/network/README.md.